src/main/webapp/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
```

Comment:
is there a way to deploy so we can try using it ?

Reply:
Yeah, I'm adding the project id to the pom.xml file. That should let me be able to deploy.